backend/src/main.py

```diff
@@ -1,25 +1,24 @@
 import asyncio
 import logging
-from hello_world.v1.greeter_rsm import Greeter
-from greeter_servicer import GreeterServicer
+from chat_servicer import ChatServicer
+from chat.v1.chat_rsm import Chat
 from resemble.aio.applications import Application
 from resemble.aio.workflows import Workflow
 
 logging.basicConfig(level=logging.INFO)
 
-EXAMPLE_GREETER_ID = 'greeter-hello-world'
+EXAMPLE_CHAT_ID = 'test'
 
 
 async def initialize(workflow: Workflow):
-    greeter = Greeter(EXAMPLE_GREETER_ID)
-
-    # Implicitly construct greeter upon first write.
-    await greeter.Greet(workflow, greeting="Hello, World!")
+    chat = Chat(EXAMPLE_CHAT_ID)
 
+    # Implicitly construct chat upon first write.
```

Comment:
This comment is no longer correct. We are using a constructor so the creation is no longer 'implicit'.